Project-1/TimPrudhomme_Project1.md

```diff
@@ -0,0 +1,483 @@
+    Tim^I		&=& t \\
+    Jill^I		&=& j \\
+    Ralph^I		&=& r \\
+    Maynard^I		&=& q \\
+    Matt^I		&=& m \\
+    Extra^I		&=& 5 \\
+    CoolestGuy^I	&=& t \\ 
+    B^I			&=& \set{(t, r), (r, q), (t, q)} \\
+    C^I			&=& \set{(m, t)} \\
+    siblingOf^I		&=& \set{(j, t), (t, q) } \\
+    Sibling^I		&=& \set{t, r} \\
+\end{array}
+\right\}
+```
+
+$I_1$ is a model of $K$ because it satisfies every axiom of $T$ and every assertion of $A$. 
+
+It's still a model even though:
+* It interprets concept names not in the knowledge base, $Extra$ and $CoolestGuy$
+* Extra elements are in the extension of $siblingOf$
+* The same element $t$ satisfies multiple concept names, $Tim$ and $CoolestGuy$, without a Unique Name Assumption preventing this
+* Axiom $Sibling \equiv ∃siblingOf$ plus the assertion that $Ralph : Sibling$ means someone is a sibling of Ralph, but no relation satisfies this. The Open World Assumption allows for incomplete knowledge about who is Ralph's sibling.
+
+---
+# Question 6
+6. Explain the difference - using natural language - between the first-order prefixes:
+```
+(a) ∃x∀y and ∀x∃y
+(b) ∃x∀y∃z and ∀x∃y∀z 
+(c) ∀x∃y∀z∃w and ∃x∀y∃z∀w
+```
+
+## Answers:
+(a) `∃x∀y` means "There exists some x such that for all y". But `∀x∃y` means "For all x, there exists some y".
+
+(b) `∃x∀y∃z` means "There exists some x such that, for all y, there exists some z". But `∀x∃y∀z` means "For all x, there exists some y such that for all z".
+
+(c) `∀x∃y∀z∃w` means "For all x there exists some y such that for all z there exists some w". But `∃x∀y∃z∀w` means "There exists some x such that for all y there exists some z such that for all w".
+
+---
+# Question 7
+7. Show that the following sentences are not equivalent by exhibiting a graph that models one but not both of these sentences:
+```
+∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))
+```
+
+## Answers:
+```mermaid
+graph LR
+0((0))
+1((1))
+2((2))
+
+0 ---> |<=| 1
+1 ---> |<=| 2
+0 ---> |<=| 2
+
+0 ---> |<=| 0
+1 ---> |<=| 1
+2 ---> |<=| 2
+```
+
+* Interpretation of `R` as less than or equal to (`<=`) with the domain `{0, 1, 2}`
+* This models `∃x∀y∃z(R(x,y) ∧ R(x,z) ∧ R(y,z))` because in this domain there's something `∃x` that is less than or equal to everything. Likewise for `∃z` but `x=z` in this model anway.
+* This doesn't model `∀x∃y∀z(R(x,y) ∧ R(x,z) ∧ R(y,z))` because not everything is less than or equal to everything (`∀x∀y(R(x,y))`): 2 is not less than or equal to 0
+
+
+---
+# Question 8
+8. Using an online tableau proof generator - such as the one found here `https://www.umsu.de/trees/` - provide tree proofs of the following entailments, which are known as the De Morgan's laws:
+```
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+(b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+(c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+(d) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+```
+
+## Answers:
+
+(a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx)) is valid
+```
+        1.¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
```

Comment:
What do you think justifies the move from 1 to 2? There is stripping of a quantifier without removing the outer negation.